### PR TITLE
added conditions component. Also fixed bug with css transition by rem…

### PR DESCRIPTION
**Pull request by joefitter**
…oving it. Added word-break to text diff

### CODE REVIEW

File: src/conditions/index.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React, { Component, Fragment } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { Form } from '../';
+import { Warning } from '@ukhomeoffice/react-components';
+
+class Conditions extends Component {
+  constructor(options) {
+    super(options);
+    this.toggleEdit = this.toggleEdit.bind(this);
```
Comment (lennym) [inline src/conditions/index.jsx]: Is there a reason to do it this way and not the arrow function way we achieve this in other places?

Comment (joefitter) [reply src/conditions/index.jsx]: yes, we don't have that babel plugin enabled in non-projects land :(

Comment (lennym) [reply src/conditions/index.jsx]: Carry on then.

Comment (joefitter) [reply src/conditions/index.jsx]: I don't think we do anyway, we didn't before.. perhaps we do? Are these my hands?

**Comment by lennym** [comment]
We can remove the styles at https://github.com/UKHomeOffice/asl-components/blob/master/src/notification/index.scss#L32-L48 too if the transitions are going.

**Comment by joefitter** [comment]
good shout - how do you feel about the transitions going? It seemed to be a minor ball ache to fix them

**Comment by lennym** [comment]
100% fine with it.

File: src/diff-text/index.jsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const DiffText = ({
   currentLabel = 'Current',
   proposedLabel = 'Proposed'
 }) => {
-  const diff = jsdiff.diffChars(oldValue, newValue);
+  const diff = jsdiff.diffWords(oldValue, newValue);
```
Comment (lennym) [inline src/diff-text/index.jsx:21]: It makes no difference if we `import { diffWords } from 'diff';` in terms of bundle size, does it?

Comment (joefitter) [reply src/diff-text/index.jsx:21]: you're RIGHT - updated

Comment (lennym) [reply src/diff-text/index.jsx:21]: I don't know what you've updated, but meh.

**Pull request merged**